components/board.upload/R/upload_module_computepgx.R

```diff
@@ -428,77 +431,58 @@ upload_module_computepgx_server <- function(
         return(ui)
       })
 
+      
       ## Checks specific for time series
       # shiny::observeEvent(samplesRT(), {
       shiny::observeEvent(
-        {
-          samplesRT()
-          countsX()
-        },
-        {
-          Y <- samplesRT()
-          Contrasts <- contrastsRT()
-          colnames(Y) <- tolower(colnames(Y))
-          Contrasts <- Contrasts[rownames(Y), , drop = FALSE]
-
-          time.var <- playbase::get_timevars()
-          sel.time <- grep(time.var, colnames(Y), ignore.case = TRUE)
-
-          if (length(sel.time) && length(unique(Y[, sel.time[1]])) > 1) {
-            timeseries <- gsub("\\D", "", unname(as.character(Y[, sel.time[1]])))
-            ia.ctx <- ia.spline.ctx <- c()
-
-            i <- 1
-            shiny::req(Contrasts)
-            for (i in 1:ncol(Contrasts)) {
-              ctx <- colnames(Contrasts)[i]
-              if (strsplit(tolower(ctx), ":")[[1]][1] %in% time.var) next
-              tt <- table(data.frame(ctx = Contrasts[, ctx], time = Y[, sel.time[1]]))
-              zeros.obs <- apply(tt, 1, function(x) sum(x == 0))
-
-              if (length(unique(timeseries)) == 1 && unique(timeseries)[1] == "") {
-                if (!any(zeros.obs)) ia.ctx <- c(ia.ctx, ctx)
-              } else {
-                if (!any(zeros.obs >= (ncol(tt) - 1))) {
-                  ia.spline.ctx <- c(ia.spline.ctx, ctx)
-                }
+      {
+        samplesRT()
+        countsX()
+      },
+      {
+        Y <- samplesRT()
+        Contrasts <- contrastsRT()
+        colnames(Y) <- tolower(colnames(Y))
+        Contrasts <- Contrasts[rownames(Y), , drop = FALSE]
+
+        time.var <- playbase::get_timevars()
+        sel.time <- grep(time.var, colnames(Y), ignore.case = TRUE)
+
+        if (length(sel.time) && length(unique(Y[, sel.time[1]])) > 1) {
+
+          timeseries <- gsub("\\D", "", unname(as.character(Y[, sel.time[1]])))
+          ia.ctx <- ia.spline.ctx <- c()
+          i <- 1
+          shiny::req(Contrasts)
+          for (i in 1:ncol(Contrasts)) {
+            ctx <- colnames(Contrasts)[i]
+            if (strsplit(tolower(ctx), ":")[[1]][1] %in% time.var) next
+            tt <- table(data.frame(ctx = Contrasts[, ctx], time = Y[, sel.time[1]]))
+            zeros.obs <- apply(tt, 1, function(x) sum(x == 0))
+            if (length(unique(timeseries)) == 1 && unique(timeseries)[1] == "") {
+              if (!any(zeros.obs)) ia.ctx <- c(ia.ctx, ctx)
+            } else {
+              if (!any(zeros.obs >= (ncol(tt) - 1))) {
+                ia.spline.ctx <- c(ia.spline.ctx, ctx)
```

Comment:
Please avoid things like that, for what I understand you just changed the indentation of this whole code, but if there is any change inside it I have no clue. This makes it very hard to review PRs